packages/desktop-client/src/components/select/DateSelect.tsx

```diff
@@ -6,10 +6,12 @@ import React, {
   useLayoutEffect,
   useImperativeHandle,
   useMemo,
+  type MutableRefObject,
+  type KeyboardEvent,
 } from 'react';
 import { useSelector } from 'react-redux';
 
-import * as d from 'date-fns';
+import { parse, parseISO, format, subDays, addDays, isValid } from 'date-fns';
```

Comment:
From
![image](https://github.com/actualbudget/actual/assets/5285928/58cf6e16-8404-4fc2-9490-0255c93e2755)

To
![image](https://github.com/actualbudget/actual/assets/5285928/ba8b926a-a3fb-4734-a69e-2d27a6a54ae6)

Reply:
Amazing